src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -13,11 +15,48 @@ const useStyles = makeStyles({
 const HealthCardDisplay = ({ patientData }) => {
   const classes = useStyles();
 
-  const immunizationCode = (codings) => {
+  const [tradenames, setTradenames] = useState({});
+
+  useEffect(() => {
+    async function fetchTradenames() {
+      const response = await axios.get(tradenamesXml, {
+        Accept: 'application/xml'
+      });
+      let data = await response.data;
+      data = data.replace(/&/g, '&amp;')
+        .replace(/"/g, '&quot;')
+        .replace(/'/g, '&apos;');
+      const parser = new DOMParser();
+      const xmlDoc = parser.parseFromString(data, 'application/xml');
+      if (xmlDoc.getElementsByTagName('parsererror').length > 0) {
+        throw new Error('Error parsing XML');
+      }
+      const prodInfos = xmlDoc.evaluate('//productnames/prodInfo',
+        xmlDoc, null, XPathResult.ORDERED_NODE_ITERATOR_TYPE, null);
+      let prodInfo = null;
+      const tn = {};
+      while ((prodInfo = prodInfos.iterateNext())) {
+        if (tn[prodInfo.children[5].textContent.trim()]) {
+          tn[prodInfo.children[5].textContent.trim()] = prodInfo.children[3].textContent
+        } else {
+          tn[prodInfo.children[5].textContent.trim()] = prodInfo.children[1].textContent
+        }
+      }
```

Comment:
Linter doesn't like that the assignment is in the `while` loop. Something like below should work.
```suggestion
      let prodInfo = prodInfos.iterateNext();
      const tn = {};
      while (prodInfo) {
        if (tn[prodInfo.children[5].textContent.trim()]) {
          tn[prodInfo.children[5].textContent.trim()] = prodInfo.children[3].textContent
        } else {
          tn[prodInfo.children[5].textContent.trim()] = prodInfo.children[1].textContent
        }
        prodInfo = prodInfos.iterateNext();
      }
```